client/state/sites/features/README.md

```diff
@@ -0,0 +1,38 @@
+# Site Features
+
+A module for managing site features data.
```

Comment:
Should we elaborate a little on what the features are, and what they can be used for?

Reply:
Tes, for sure. Could we do that in a follow-up PR?

Reply:
Docummenting now